elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/core/QueryLogger.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.core;
+
+import com.yahoo.elide.datastores.aggregation.query.Query;
+
+import java.security.Principal;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import javax.ws.rs.core.MultivaluedMap;
+
+/**
+ * Query Logger Interface for Elide
+ */
+public interface QueryLogger {
+
+    /**
+     * Accepts the incoming GraphQL query and notes the start time for the query
+     * @param queryId The RequestScope requestId.
+     * @param user The Principal user
+     * @param headers Http Request Headers
+     * @param apiVer API Version
+     * @param apiQuery QueryString of the requested URL request
+     */
+    void acceptQuery(UUID queryId, Principal user, Map<String, String> headers,
```

Comment:
Do we need both of these acceptQuery methods or can we just have the one below?

Reply:
We can just have the one below. Do not need the other one since Querylogger no longer lives in Elide. Will update this